examples/bert/bert_model.py

```diff
@@ -546,10 +482,10 @@ def __init__(
             name="word_embeddings",
         )
 
-        self._position_embedding_layer = PositionEmbedding(
+        self._positional_embedding_layer = keras_nlp.layers.PositionalEmbedding(
             initializer=initializer,
             max_length=max_sequence_length,
-            name="position_embedding",
+            name="positional_embedding",
```

Comment:
Will this renaming prevent us from reloading models saved with the old names?

Reply:
Agreed, no need to worry about any compat at this point.

Reply:
Based on the naming discussion, this will be reverted to `position_embedding` anyway.